pkg/k8sutil/k8sutil.go

```diff
@@ -32,6 +32,15 @@ func GetWatchNamespace() (string, error) {
 	return ns, nil
 }
 
+// GetNamespace returns the namespace the operator should be running in.
+func GetNamespace() (string, error) {
```

Comment:
GetNamespace is a very generic name, could we name it something like `GetClusterNamespace`. Also not sure we need to export this?